tests/virtualization/universal/prepare_guests.pm

```diff
@@ -91,6 +91,11 @@ sub run {
 
     # Ensure additional package is installed
     zypper_call '-t in libvirt-client iputils nmap supportutils';
+
+    # If testing libvirt package specificly
+    if ('libvirt' eq get_var("UPDATE_PACKAGE")) {
+        zypper_call '-t in libvirt';
```

Comment:
This seems the easiest way to make automation pass. But is it meanfingful in real customer scenarios to check if this package is updated to incident repo? Just as developer explained in the bug, `libvirt` package is optional to be installed when setting up kvm host. It is not required in package dependency. So in real host, `libvirt` package is likely not installed either. What's really meaningful for libvirt related incidents, I am afraid, is to check libvirt-damon-xx and other  required packages for virtualization host. HDYT?

Reply:
As discussed in the poo#116407, libvirt is "meta" package that installs more dependencies then the standard xen/kvm patterns from YaST. I think when a new version of the libvirt package is release it makes sens to install it "fully" when testing even that will be "bloated" for production server.

Reply:
I read the ticket poo#116407 fully. I am afraid this solution is not fully discussed and agreed. See https://progress.opensuse.org/issues/116407#note-20.